src/documentation/pages/components/forms/range-slider/examples/_base.vue

```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  <ComponentExample title="Base" id="base" padding="-p--0" :tabs="exampleTabs">
```

Comment:
i don't think padding="-p--0" is needed here, nor the p--3 below since the default padding is already -p--3, so you don't have to add or remove any padding for this example and similar ones